portfolio/src/main/webapp/script.js

```diff
@@ -125,19 +123,46 @@ function getOrder() {
 /**
  * Checks whether the user is logged in or not. If the user is logged in, then
  * display the comments section. Otherwise, the comments section will remain
- * hidden.
+ * hidden, and the user is directed to a login page.
  */
 function isLoggedIn() {
-  fetch('/login').then(response => response.text()).then((userLoggedIn) => {
-    console.log('User login status: ' + userLoggedIn);
+
+  fetch('/login').then(response => response.json()).then((status) => {
```

Comment:
As discussed in PR18, status is reserved for HTML so should be avoided as a variable name.  Since you can't use 'status', prefer a longer, more descriptive name, as opposed to a shortening like 'stat'.